beta-component-promotions/2021.06-promotions/2021.06-promotions.md

```diff
@@ -0,0 +1,42 @@
+# PatternFly Beta Component Promotion Candidates
+## For Release 2021.08
+### [Banner](https://www.patternfly.org/v4/components/banner)
+A banner is a one-line, full color, full width container that can be used to communicate short snippets of information to users. Banners are un-intrusive, non-dismissible, and only support link buttons.
```

Comment:
```suggestion
A **banner** is a one-line, full-color, full-width container that can be used to communicate short snippets of information to users. Banners are un-intrusive and non-dismissible, and they only support link buttons.
```

Reply:
Added bolding for component names to align with the bolding formatting we use in design guidelines. Also, should "un-intrusive" be "nonintrusive" ? I keep seeing the latter come up in Google searchers.